packaging/makeself/install-or-update.sh

```diff
@@ -57,6 +57,10 @@ while [ "${1}" ]; do
   shift 1
 done
 
+if [ x"$DO_NOT_TRACK" = x"1" ]; then
```

Comment:
Oh I just remembered that we only need `DO_NOT_TRACK` set to a non-empty string, not to any particular value. See below:

https://github.com/netdata/netdata/blob/fa92c1e3b4990d76a87bb6d85297c954b667f397/packaging/docker/Dockerfile#L66

https://github.com/netdata/netdata/blob/fa92c1e3b4990d76a87bb6d85297c954b667f397/packaging/docker/run.sh#L10

https://github.com/netdata/netdata/pull/7437#issuecomment-561937299

Reply:
I _read_ this as `export DO_NOT_TRACK=1` elsewhere on the Internet 🤣

Wo which is it? :)

Which should we support? One form, both? I opted for making it consistently `DO_NOT_TRACK=1` everywhere now.

Reply:
This is done by one dude and it's not really a standard or anything, so I'm not fussed which one. The [comment I linked](https://github.com/netdata/netdata/pull/7437#issuecomment-561937299) is the author himself saying it should work with any value.

Reply:
Ahh okay; I'll buy it. Give me a few mins to amend the PRs :)

Reply:
Done